### PR TITLE
Add a warning on the Scratch forums to avoid posts about Scratch Addons

### LINKED ISSUE #552: Add a warning on the Scratch forums to avoid posts about Scratch Addons
There are still people who report bugs on the Bugs and Glitches category/subforum. I think we should add either an addon or just an extension feature so there will be a warning to stop people post about Scratch Addons on that category. Maybe also do the same to the Suggestion category/subforum and griffpatch's comment section, if you want.

### PR DESCRIPTION
**Pull request by TheColaber**
**Resolves**

Resolves #552

**Changes**

I've added this warning to the forum when posting:
image

**Reason for changes**

It's pretty common...

**Tests**

Well it works on chrome. 

Btw Please someone tell me how to run prettier. I need to know...

### CODE REVIEW

**Comment by pufferfish101007** [comment]
To run prettier go to the actions tab on your fork and vlick enable actions or donething like that. Then, whenever a commit is made, prettier will run, giving you syntax errors if there are any and fogmatting all your code.

**Comment by GrahamSH-LLK** [comment]
Please make this only show on Bugs and Glitches. You can check the href to see if it is the correct id.

**Comment by TheColaber** [comment]
> Please make this only show on Bugs and Glitches. You can check the href to see if it is the correct id.

Ok, Also, it won't be the href that I should check, because it looks like this:
`https://scratch.mit.edu/discuss/topic/435880/`

**Comment by mxmou** [comment]
@TheColaber It's supposed to look like this: https://scratch.mit.edu/discuss/3/topic/add/

**Comment by TheColaber** [comment]
> @TheColaber It's supposed to look like this: https://scratch.mit.edu/discuss/3/topic/add/

Oh yeah lol...

File: content-scripts/cs.js [hunk 1]
```diff
@@ -142,3 +143,35 @@ function onHeadAvailable({ globalState, addonsWithUserscripts, userstyleUrls, th
   });
   observer.observe(template, { attributes: true });
 }
+
+function forumWarning() {
+  if (window.location.pathname == "/discuss/3/topic/add/") {
```
Comment (WorldLanguages) [inline content-scripts/cs.js:148]: You should move this if to line 39, and remove it from here

**Pull request merged**